### DISCUSSION TITLE
AOC Day 18: Geometry Wars

### DISCUSSION THREAD

**Discussion by rhdeck** [Virtual-Coffee/virtualcoffee.io#784] [Advent of Code 2022]
Discussion of solutions for [Advent of Code Day 18](https://adventofcode.com/2022/day/18) here.

Don't forget about the VC Advent of Code private leaderboard:

Go to [leaderboard/private](https://adventofcode.com/2022/leaderboard/private)
Use join code 363496-e6f42283

**Comment by rhdeck** [comment]
I had a much harder time working on this. Spent a lot of time debugging a solution to part 2 that would "stack" on my solution from part 1, but couldn't get a right answer. Tossed it in favor of building a 3-d mold maker. [I kinda like it!](https://github.com/rhdeck/aoc-2022/blob/main/src/18/day_18.ipynb)

**Comment by doleraj** [comment]
https://github.com/doleraj/adventofcode/blob/main/aoc2022/src/day18/index.ts

Bleh. BLEH. I ended up doing a ton of work assuming that I didn't need to reach a -1 index and so I just had to redo all of it.
